.github/workflows/non-english-warning.yml

```diff
@@ -0,0 +1,45 @@
+name: Check for non-English updates outside of Crowdin
+
+on:
+  pull_request:
+    paths:
+      - "public/content/translations/**/*.md"
+      - "src/intl/**/*.json"
+      - "!src/intl/en/**
+
+jobs:
+  check_branch_name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Exit early if branch name contains 'crowdin'
+        run: |
+          if [[ "${{ github.head_ref }}" == *crowdin* ]]; then
+            echo "Branch name contains 'crowdin', stopping workflow"
+            exit 1
+          fi
+
+  add_label_and_comment:
+    needs: check_branch_name
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add label and comment
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const prNumber = context.issue.number;
```

Comment:
(GitHub treats PR's as a type of issue)

Reply:
`context` made available from the Octokit package, which is automatically made available with `actions/github-script@v5`